### ISSUE TITLE
Benchmark encryption & combination (TypeScript via WASM bindings) and partial decryption (Python bindings)

### ISSUE THREAD

**Issue by cygnusv** [nucypher/ferveo#11]

**Comment by piotr-roslaniec** [comment]
WASM bindings benchmark
----------------------------------------
* Results available [here](https://github.com/piotr-roslaniec/ferveo/blob/c0c2c9e21f65eb2e6cf3a58b9952a3d9b0bb38f6/tpke-wasm/BENCHMARK.md)
* WASM in the browser is 14x slower than Rust on desktop
* The worst-case scenario performance-wise is where we use 128 shares- Time to combine shares is in the ballpark of a few seconds (2.2s in the benchmark)
* **TLDR: Combining shares takes up to ~2-3s in the browser.**

Relevant PR: https://github.com/piotr-roslaniec/ferveo/pull/2

**Comment by piotr-roslaniec** [comment]
Python bindings benchmark
-----------------------------------------
- Results available [here](https://github.com/piotr-roslaniec/ferveo/blob/0091ae1f8521337967a6dbb90157875a17fff7f9/tpke-python/BENCHMARK.md#results)
- Notice: The code used in the benchmark doesn't perform [public verification](https://nikkolasg.github.io/ferveo/tpke.html#public-verification-of-decryption-shares) yet. See this [question](https://github.com/nucypher/ferveo/issues/5#issuecomment-1303244116) about "simple vs fast" decryption if it impacts how verification is performed.
- **TLDR: Partial decryption of 128 shares takes up to 25 ms in Python**

**Comment by piotr-roslaniec** [comment]
Benchmarks are done. Please reopen should we need to update the benchmark suite.
